packages/components/src/Popover/Layout/PopoverHeader/PopoverHeader.tsx

```diff
@@ -50,19 +52,27 @@ type DetailOptions = WithDetail | WithHideClose
 
 export type PopoverHeaderProps = DetailOptions & {
   children: ReactNode
+  headerVisuallyHidden?: boolean
```

Comment:
Given that this is in the context of `PopoverHeader` already this could be shortened a good bit.

```suggestion
  hidden?: boolean
```

Make sure to add a JSDoc explaining the use-case.